devices/stm32l4x3.yaml

```
@@ -11,6 +11,21 @@ RCC:
       USBF:
         name: USBFSEN
         description: USB FS clock enable
+        
+# SVD incorrectly shifts CRCEN/CRCRST/CRCSMEN  11 bits instead of 12
+RCC:
```

Currently this duplicates the RCC key in the YAML, so the previous patch doesn't get applied, which knocks out the USFB -> USBFSEN fix. Just need to remove `RCC:` to fix it up.

```suggestion
        
    # SVD incorrectly shifts CRCEN/CRCRST/CRCSMEN  11 bits instead of 12
```